docs/integrations/aws.md

```diff
@@ -435,6 +435,25 @@ For the above example, the objects in S3 will be saved with tags: `my_key1=my_va
 
 For more details on tag restrictions, please refer [User-Defined Tag Restrictions](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/allocation-tag-restrictions.html).
 
+### S3 Access Points
+
+[Access Points](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-access-points.html) can be used for operating
+s3 objects from the specified bucket, against which access-point was mapped. This is useful for multi-region access,
```

Comment:
Thanks @singhpk234 for working on this. It is really helpful. Wondering if we can elaborate more on how multi-region access is enabled by this feature. I assume we just need to clarify that one access point can access multiple buckets cross-regions. For the below example, we can add a spark-sql config with the different bucket name and the same access point, so that user can understand better how it works.

Reply:
>  I assume we just need to clarify that one access point can access multiple buckets cross-regions. For the below example, we can add a spark-sql config with the different bucket name and the same access point, so that user can understand better how it works.

Makes sense, have made the changes, Thanks @flyrain :) !!!